test/valid-seq-json/note-valid.json

```diff
@@ -0,0 +1,35 @@
+{
+  "id": "note",
+  "metadata": {},
+  "steps": [
+    {
+      "description": "Note step with all possible fields.",
+      "models": [
+        {
+          "offset": "00:00:00.000",
+          "variable": "model_var_float",
+          "value": "1.234"
+        },
+        {
+          "offset": "00:00:00.001",
+          "variable": "model_var_int",
+          "value": "-1234"
+        },
+        {
+          "offset": "01:02:03.000",
+          "variable": "model_var_string",
+          "value": "Model test string"
+        },
+        {
+          "offset": "10:00:00.000",
+          "variable": "model_var_uint",
+          "value": "1234"
+        }
+      ],
+      "string_arg": "Note Test.",
+      "time": { "type": "BLOCK_RELATIVE" },
```

Comment:
This needs a `tag` value, right?

Reply:
It does! Good point... Not sure why it passed validation

Reply:
It passed validation because our schema doesn't actually have logic to enforce that `time` have the `tag` field since `COMMAND_COMPLETE` doesn't require it. Maybe we take this opportunity to improve that, too.

Reply:
I went ahead and updated the validation to check for presence of `tag` for everything but command_complete, and added some tests to that effect as well.

Reply:
And updated this test to include a tag!